pkgs/gnolang/precompile.go

```diff
@@ -199,6 +203,22 @@ func PrecompileBuildPackage(fileOrPkg string, goBinary string) error {
 	return nil
 }
 
+func guessRootDir(fileOrPkg string, goBinary string) (string, error) {
+	abs, err := filepath.Abs(fileOrPkg)
+	if err != nil {
+		return "", err
+	}
+	args := []string{"list", "-m", "-mod=mod", "-f", "{{.Dir}}", "github.com/gnolang/gno"}
+	cmd := exec.Command(goBinary, args...)
+	cmd.Dir = abs
+	out, err := cmd.CombinedOutput()
+	if err != nil {
+		return "", fmt.Errorf("can't guess --root-dir")
```

Comment:
You don't need to use `fmt.Errorf` here if you are not formatting an error - use `errors.New` instead, or add the error to the output error message in the formatted string